src/modules/commander/commander.cpp

```diff
@@ -1731,6 +1733,9 @@ int commander_thread_main(int argc, char *argv[])
 									(hrt_abstime)disarm_when_landed * 1000000);
 			}
 
+			int32_t v;
+			param_get(_param_acro_att, &v);
+			acro_att = (v != 0);
```

Comment:
No, there is no boolean param. According to `src/modules/systemlib/param/param.h` there are only 3 types: int32, float and structure, but `Tools/px_generate_params.py` supports only INT32 and FLOAT.
You can mimic boolean with enum:
```
 * @value 0 Classic Acro Mode
 * @value 1 Attitude Acro Mode
```
And in code define simple enum class with matching values.